### ISSUE TITLE
Is this issue now fixed upstream?

### ISSUE THREAD

**Issue by davelab6** [HanFox/AdventPro#1]
Given the licensing issue you described, I wonder if this repo is now deprecated by the upstream? :)

**Comment by HanFox** [comment]
You are right. Apparently it was finally updated on Google Fonts on December 2018.

I'll delete this repo in the coming days.

**Comment by HanFox** [comment]
Oh, no, kerning between capital Q and period and capital P and period is still broken...
